joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/querying/ast/InheritanceFullNameTests.scala

```diff
@@ -0,0 +1,103 @@
+package io.joern.csharpsrc2cpg.querying.ast
+
+import io.joern.csharpsrc2cpg.testfixtures.CSharpCode2CpgFixture
+import io.shiftleft.semanticcpg.language.*
+
+class InheritanceFullNameTests extends CSharpCode2CpgFixture {
+  "inherited type full names from classes" should {
+    val cpg = code("""
+        |public class Foo {
+        |    public string Bar {get; set;}
+        |}
+        |
+        |public class Bar: Foo {}
+        |
+        |public class Baz {
+        | public void main() {
+        |   Bar b = new Bar();
+        | }
+        |}
+        |""".stripMargin)
+
+    "resolve full names for classes with inheritance" in {
+      inside(cpg.typeDecl.nameExact("Bar").l) {
+        case bar :: Nil =>
+          bar.fullName shouldBe "Bar"
+          bar.inheritsFromTypeFullName shouldBe Seq("Foo")
```

Comment:
`Foo`'s full name is equal to its short name. Could you wrap this in some namespace to make sure it's a full name, please?

Reply:
Similarly on other tests below